packages/shared-ux/page/analytics_no_data/src/analytics_no_data_page.component.test.tsx

```diff
@@ -7,20 +7,31 @@
  */
 
 import React from 'react';
+import { act } from 'react-dom/test-utils';
+
 import { mountWithIntl } from '@kbn/test-jest-helpers';
-import { KibanaNoDataPage } from '@kbn/shared-ux-components';
+import { KibanaNoDataPage } from '@kbn/shared-ux-page-kibana-no-data';
+
 import { AnalyticsNoDataPage } from './analytics_no_data_page.component';
+import { AnalyticsNoDataPageProvider } from './services';
+import { getJestServices } from './mocks';
 
 describe('AnalyticsNoDataPageComponent', () => {
+  const services = getJestServices();
   const onDataViewCreated = jest.fn();
 
-  it('renders correctly', () => {
+  it('renders correctly', async () => {
```

Comment:
Should this test case be moved to `analytics_no_data_page.test` then? And this whole file can be removed if not needed? I prefer to not have a test with services for a pure component.

Reply:
I should clarify: the services have to be there because of the _inner_ components.  With no context, inner components won't render.

Reply:
My question is that this test is very similar to this one: https://github.com/elastic/kibana/pull/133593/files#diff-02ddbbd43a61965c22ef73f78c8e6166f8f1efa7283ec63a267f3092ea7fff2bL39

I don't think we need two tests for almost the same thing?